packages/front-end/components/FactTables/FactMetricModal.tsx

```diff
@@ -220,12 +281,46 @@ function ColumnRefSelector({
             required
           />
         </div>
+        {inlineFilterFields.map(({ label, key, options, error }) => (
+          <div className="col-auto" key={key}>
+            <MultiSelectField
```

Comment:
WDYT about formatting this so as soon as you select a second value, we write "is any of" after the column name (e.g. "browser is any of")? 

... or, put "any of" somewhere more attractive to signify that it's going to be an OR
... or, even write IN( multi-select field here ), but that's clunkier.

Reply:
I can put a similar tooltip as the Additional Filters input which says how multiple values are treated.